docs/Reference/Certificates/generateProvisioningCert.md

```diff
@@ -0,0 +1,135 @@
+--8<-- "References/abbreviations.md"
+	
+## Why generate a customer provisioning certifcate?
```

Comment:
Recommend customers to use https://open-amt-cloud-toolkit.github.io/docs/2.13/GetStarted/createProfileACM/#what-youll-need, because custom provisioning certificate flow will require them to touch the device (which is adding hash to MEBx)